src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
Talking down to someone with statements like; let me boil it down to basic, so you can get it and such.  Yes those kinds of statements that you made are insults.  I wasn't born yesterday.  I know what those statements are about.  They are intended to belittle.

I'm not the one arguing in circles and ignoring your explanations.  I'm debunking them and the reasoning that's being given for them.